profiling/src/profile/pprof.rs

```diff
@@ -117,7 +117,7 @@ pub struct Mapping {
     #[prost(bool, tag = "8")]
     pub has_filenames: bool,
     #[prost(bool, tag = "9")]
-    pub has_line_numbers: bool,
+    pub has_lines: bool,
     #[prost(bool, tag = "10")]
```

Comment:
‼ This change seems incorrect, since the pprof format shouldn't change itself...? (Maybe too much search&replace?)